umh.docs.umh.app/content/en/docs/getstarted/installation.md

```diff
@@ -56,38 +56,36 @@ please refer to our dedicated [Firewall Rules](/docs/production-guide/security/f
 3. If you are not a member, continue with sign up. Register your information and click on **SIGN UP**.
    ![Sign up page](/images/getstarted/installation/signup.png)
 
-4. Click on **+Add Instance** button.
-   ![Add Instance button](/images/getstarted/installation/dashboard.png)
+4. You should've been redirected to the **Network View**. Now click on the **+Add Instance** button at the top right corner.
 
-5. Select **Install UMH Only**.
-   ![Install UMH Only button](/images/getstarted/installation/addinstance.png)
+5. Go through the installation wizard by first clicking on **Install the UMH** and then on **UMH Classic**.
 
-6. Enter your instace name and then click on **Create my command**.
-   ![Instance name input](/images/getstarted/installation/entername.png)
+6. Configure the instance to your needs. The only required field is the **Instance Name**.
+   Optionally you can define the location of the instance according to the ISA-95 standard.
```

Comment:
what happens if one does it? so what is the result and why should one do it?

Something like this:
In order to keep multiple UMH instancers ordered, you could assign them to an enterprise, site, etc. in the ISA95 model. THis also has the advantage, that the system will be able to identify misconfigured topics. So if you add a UMH instance to a site called ABC, but then the UNS of that instance has messages for site ACB (probably a spelling mistake), it will highlight them. 

EIther explain it here, or maybe explain it in detail in a feature description, and then link to that feature description for more information